model_api/python/openvino/model_api/adapters/openvino_adapter.py

```diff
@@ -18,13 +18,16 @@
 from pathlib import Path
 from typing import Dict, Set, Tuple
 
+import onnx
```

Comment:
Please, move the import under `if Path(self.model_path).suffix == ".onnx" and not weights_path:` to make it optional

Reply:
Done